static/css/mkp.scss

```diff
@@ -1,5 +1,13 @@
 ---
 ---
+
+footer {
+	.row {
+		padding: 24px 0;
+		border-top: 2px solid #888;
```

Comment:
can we add #888 as a constant in template.yml? I'm mostly wondering if I need to use it for other borders.

Reply:
Changed to the color called 'muted', it was 777.